src/providers/ThemeProvider/ThemeProvider.js

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useState, useEffect, useMemo, useContext } from 'react';
+
```

Comment:
Suggestion: In dark mode contrast ratio could be improved for readability
![image](https://user-images.githubusercontent.com/91944618/159805646-b554833b-9af0-4e8a-a5c6-1a6c2e0c8689.png)